### PR TITLE
Unfollow (and whitelist change)

### PR DESCRIPTION
**Pull request by debugwand**

### CODE REVIEW

**Comment by debugwand** [comment]
tests have passed but circle/github is not updating the orange dot :(

File: server/v2/controllers/relationship/replace.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+"use strict";
+const crud = require('../../models/crud');
+const whitelist = require('../../lib/whitelist');
+const purgeCache = require('../../lib/purge-cache');
+
+module.exports = (req, res, next) => {
+	const node = 'user';
+	const type = 'concept';
```
Comment (laurieboyes) [inline server/v2/controllers/relationship/replace.js]: Are these values supposed to be hard-coded?

Comment (debugwand) [inline server/v2/controllers/relationship/replace.js]: the only way I could think of to handle this was to pick out the POST and DELETE requests to /user/:uuid/followed/concept, so they no longer come from the params in the routing.

Comment (laurieboyes) [inline server/v2/controllers/relationship/replace.js]: Ah right yeah I see. Could we rename the file to something less generic? The only thing I don't like about it is that on the face of it it looks like a generic relationship-replace module

**Pull request merged**